packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -203,6 +203,12 @@ export default {
             default: 1000
         },
 
+        spinnerTimeout: {
+            type: Number,
+            required: false,
+            default: 300
```

Comment:
This is very subjective, but should we increase this a bit, to 500ms maybe? I just want to avoid a flashy spinner when it's still quite fast, you know?

Reply:
@SachaZvetelman that's why its configurable, we can decide what to set it to in CoreWeb itself. 300ms is just something I talked about with Charlie

Reply:
@SachaZvetelman But as it's simple enough, changed to 500ms ;)